stored_requests/config/config.go

```diff
@@ -99,11 +106,16 @@ func newCache(cfg *config.StoredRequests) stored_requests.Cache {
 		return &nil_cache.NilCache{}
 	}
 
-	glog.Infof("Using a Stored Request in-memory cache. Max size for StoredRequests: %d bytes. Max size for Stored Imps: %d bytes. TTL: %d seconds.", cfg.InMemoryCache.RequestCacheSize, cfg.InMemoryCache.ImpCacheSize, cfg.InMemoryCache.TTL)
+	if cfg.InMemoryCache.RequestCacheSize <= 0 && cfg.InMemoryCache.ImpCacheSize <= 0 && cfg.InMemoryCache.TTL <= 0 {
```

Comment:
Should these log messages be in `memory.NewCache()` where the actual logic is implemented, rather than replicating that logic in this piece of code just for the purpose of logging this message?

Reply:
Much cleaner this way, yeah.

Also improves the log messages, since I can split the lines describing Stored Requests and Stored Imps.